packages/daemon/src/formula-identifier.js

```diff
@@ -12,7 +12,13 @@ export const parseFormulaIdentifier = formulaIdentifier => {
     );
   }
 
-  const type = formulaIdentifier.slice(0, delimiterIndex);
-  const number = formulaIdentifier.slice(delimiterIndex + 1);
-  return { type, number };
+  const [type, number, location] = formulaIdentifier.split(':');
```

Comment:
I’m guessing this eventually evolves to the `endo://NODE/VALUE?type=TYPE` subset of `endo://NODE/VALUE?type=TYPE&subtype=&address=&address=`